R/summary.R

```diff
@@ -25,7 +25,7 @@ summary.corpus <- function(object, n = 100, tolower = FALSE, showmeta = TRUE, ..
     ndoc_all <- ndoc(object)
     object <- head(object, n)
     ndoc_show <- ndoc(object)
-    result <- textstat_summary(object, cache = FALSE, tolower = tolower, ...)
+    result <- summarize(as.corpus(object), cache = FALSE, tolower = tolower, ...)
```

Comment:
Let's make summary to be like 
```
> summary(attr(data_corpus_inaugural, "docvars"))
   docname_                     docid_       segid_       Year     
 Length:58          1789-Washington: 1   Min.   :1   Min.   :1789  
 Class :character   1793-Washington: 1   1st Qu.:1   1st Qu.:1846  
 Mode  :character   1797-Adams     : 1   Median :1   Median :1903  
                    1801-Jefferson : 1   Mean   :1   Mean   :1903  
                    1805-Jefferson : 1   3rd Qu.:1   3rd Qu.:1960  
                    1809-Madison   : 1   Max.   :1   Max.   :2017  
                    (Other)        :52                             
  President          FirstName                           Party   
 Length:58          Length:58          Democratic           :21  
 Class :character   Class :character   Democratic-Republican: 7  
 Mode  :character   Mode  :character   Federalist           : 1  
                                       none                 : 2  
                                       Republican           :24  
                                       Whig                 : 3  
```

Reply:
The key for the separation is how to wean it from `textstat_summary()`. How we design the summary is better discussed and resolved via #1959.

Reply:
Duplication is not separation: there are `quanteda::summarize()` and `quanteda.textstats::summarize()`. I am fine if this is only transitional.

Reply:
Agreed. We will eliminate the duplication by redesigning `summary.corpus()`.